docs/developer-guide/get-started/emt-build-and-deploy.md

```diff
@@ -0,0 +1,373 @@
+
+   ```bash
+   cd edge-microvisor-toolkit
+   ```
+
+3. Install prerequisites:
+
+   > **Note:** Requirements for building the toolchain were validated on Ubuntu 22.04 host.
+
+   ```bash
+   # Install required dependencies.
+   sudo ./toolkit/docs/building/prerequisites-ubuntu.sh
+
+   # Also supported:
+   #    make -C toolkit install-prereqs
+
+   # Fix go 1.21 link
+   sudo ln -vsf /usr/lib/go-1.21/bin/go /usr/bin/go
+   sudo ln -vsf /usr/lib/go-1.21/bin/gofmt /usr/bin/gofmt
+
+   # Install and configure Docker.
+   curl -fsSL https://get.docker.com -o get-docker.sh
+   sudo sh get-docker.sh
+   sudo usermod -aG docker $USER
+   ```
+
+   **You will need to log out and log back in** for user changes to take effect.
+
+4. Navigate to the `toolkit` subdirectory.
+
+   ```bash
+   cd ./toolkit
+   ```
+
+5. Build the toolchain.
+
+   ```bash
+   sudo make toolchain REBUILD_TOOLS=y
+   ```
+
+6. Customize the toolkit image.
+
+   You can include additional components to your toolkit image by adding RPM packages.
+   For more details, see
+   [Customize Your Edge Microvisor Toolkit Image](./emt-building-howto.md#customize-your-edge-microvisor-toolkit-image).
+
+7. Build the toolkit image:
+
+   All available image configurations are stored in JSON files, located in the `imageconfigs` folder.
+   For example, to build a RAW image without real-time extensions, use `edge-image.json` and run
+   the following command:
+
+   ```bash
+   sudo make image -j8 REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/edge-image.json
+   ```
+
+   The built image will be located in its separate directory under  `edge-microvisor-toolkit/out/images/`.
+
+   For more information about specific building parameters, refer to [this article](https://github.com/open-edge-platform/edge-microvisor-toolkit/blob/3.0/toolkit/docs/building/building.md#local-build-variables).
+
+## 2. Create a bootable USB installer
+
+### Prepare the bootable USB drive
+
+Ensure the correct USB drive is selected to avoid data loss.
+Replace `/dev/sdX` with the device name of your USB drive.
+
+- Insert the USB drive and identify the USB disk:
+
+   ```bash
+   lsblk -o NAME,MAJ:MIN,RM,SIZE,RO,FSTYPE,MOUNTPOINT,MODEL
+   ```
+
+- Use the `wipefs` command to remove any existing filesystem signatures from the USB drive.
+  This ensures a clean slate for formatting
+
+   ```bash
+   sudo wipefs --all --force /dev/sdX
+   ```
+
+- Format the USB drive with the FAT32 filesystem, using `mkfs.vfat`:
+
+   ```bash
+   sudo mkfs.vfat /dev/sdX
+   ```
+
+- Unmount the USB drive to ensure the creation of bootable USB.
+
+  - Check what is currently mounted:
+
+    ```bash
+    df -hT
+    ```
+
+  - Unmount the drive:
+
+    ```bash
+    sudo umount /dev/sdX
+    ```
+
+### Prepare the installation files
+
+1. Clone the edge-microvisor-toolkit-standalone-node repository:
+
+   ```bash
+   cd
+   git clone https://github.com/open-edge-platform/edge-microvisor-toolkit-standalone-node
```

Comment:
You should have similar cloning of a tagged release as opposed to the tip of the tree for consistence.